pkg/tools/serialize/serialize_test.go

```diff
@@ -138,3 +139,20 @@ func TestAsyncExecOneAtATime(t *testing.T) {
 		<-done
 	}
 }
+
+func BenchmarkExecutorAsync(b *testing.B) {
```

Comment:
As I know `gotestsum` will not run bench tests on CI. Is it expected?

Reply:
I'd love to get to running Benchmarks on CI... just not sure what to *do* with them yet in CI  :)
Was using it locally to benchmark differences in implementation :)  On my laptop I'm averaging about 226 ns/op.  Since the function being tested itself is about 20 ns/op... I'm reading this as being basically a ~200ns tax on the function execution... which isn't bad ;)

Reply:
>I'd love to get to running Benchmarks on CI... just not sure what to do with them yet in CI :)

I've created the issue for adding bench tests on CI https://github.com/networkservicemesh/sdk/issues/117.